chainio/clients/fireblocks/contract_call.go

```diff
@@ -43,11 +43,17 @@ type ContractCallRequest struct {
 	// In case a transaction is stuck, specify the hash of the stuck transaction to replace it
 	// by this transaction with a higher fee, or to replace it with this transaction with a zero fee and drop it from the blockchain.
 	ReplaceTxByHash string `json:"replaceTxByHash"`
+	gasLimit        string `json:"gasLimit"`
+	// MaxFee and PriorityFee are the maximum and priority fees for the transaction.
+	// If the transaction is stuck, the Fireblocks platform will replace the transaction with a new one with a higher fee.
+	// These fields are required if FeeLevel is not specified.
+	MaxFee      string `json:"maxFee"`
+	PriorityFee string `json:"priorityFee"`
```

Comment:
shouldn't this be MaxPriorityFee? See eip1559 https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md

See also https://ethereum.stackexchange.com/a/134950/85362 for why this needs to be max and not fixed.

Reply:
Note that these fields are being used to call a Fireblocks API endpoint, not ETH client.
So the field names don't exactly match those defined on ETH client: https://developers.fireblocks.com/reference/post_transactions
The `priorityFee` field matches what you call maxPriorityFee.
Afaik, there's not even a way to set just "priority fee" on a transaction